phylogenetics.yaml

```diff
@@ -66,14 +65,12 @@ tools:
   tool_panel_section_label: "Phylogeny construction"
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
   install_resolver_dependencies: True
-  install_tool_dependencies: False
```

Comment:
Since 30 seconds this version should be conda ready :)
https://toolshed.g2.bx.psu.edu/view/rnateam/mafft/15974dd17515